force-app/main/default/classes/CloseOldSTBCases_Batch.cls

```diff
@@ -0,0 +1,43 @@
+global class CloseOldSTBCases_Batch implements Database.Batchable<sObject> {
+    private static LoggerUtility logger = new LoggerUtility('CloseOldSTBCases_Batch');
+    private String query = 'SELECT Id, CRM_isActive__c FROM Thread__c WHERE RecordType.DeveloperName = \'NKS_Thread\' AND CRM_Thread_Type__c = \'STB\' AND CRM_Latest_Message_Datetime__c <= N_DAYS_AGO:60 AND CRM_isActive__c = TRUE';
+
```

Comment:
Å bruke RecordType her blir feil, det vil returnere alt for få tråder, jeg kjørte queryen med og uten recordType og det førte til mange færre resultater

Reply:
Nice catch :D Implementerte jo NKS Thread recordtypen for ikke så lenge siden, hehe.